setup.cfg

```diff
@@ -20,6 +20,9 @@ addopts =
 filterwarnings =
     ignore:the matrix subclass:PendingDeprecationWarning
 
+    # Workaround for https://github.com/pypa/setuptools/issues/2885
+    ignore::DeprecationWarning:pkg_resources
```

Comment:
```suggestion
    ignore:Creating a LegacyVersion has been deprecated:DeprecationWarning
```

maybe?

Reply:
I think I understand: when `CHECK_WARNINGS='true'` the default pytest config from `setup.cfg` no longer applies.

Reply:
and indeed when we have `-Werror::DeprecationWarning` in the commandline, we also need `-Wignore:Creating\ a\ LegacyVersion:DeprecationWarning` in the commandline. `-Wignore::DeprecationWarning:pkg_resources` does not work, probably because it does not subsume `error::DeprecationWarning` in the pytest logic.

Reply:
interesting, confusing!